components/ColumnIssueCard.vue

```diff
@@ -13,5 +21,15 @@ defineProps<{ id: string; title: string }>()
     <h4>
       <NuxtLink href="#" class="w-fit text-sm hover:underline">{{ title }}</NuxtLink>
     </h4>
+
+    <ul v-if="showConfigDataLabels">
+      <li
+        v-for="label in dataLabels"
+        :key="label"
+        class="mt-2 text-xs font-bold text-rad-foreground-emphasized"
```

Comment:
Used very basic styles here since this is only for debugging purposes.

Reply:
Data labels should be shown sometimes. Non-data labels should be shown always.

Reply:
Oh, I thought showing non-data labels wasn't in the scope of this ticket.
I suggest we display the rest of the labels in a future PR to help merge this one faster. (mostly worried about the back and forth on label styling)

Reply:
Won't they look the same? Anyways, I thought it would be straightforward to show them all by default, if not easier than filtering them sometimes.
Up to you.